Library/Formula/llvm.rb

```diff
@@ -116,18 +151,74 @@ def install
     # Apple's libstdc++ is too old to build LLVM
     ENV.libcxx if ENV.compiler == :clang
 
-    if build.with?("lldb") && build.without?("clang")
-      raise "Building LLDB needs Clang support library."
+    if build.with? "clang"
+      (buildpath/"tools/clang").install resource("clang")
     end
 
-    if build.with? "clang"
+    if build.with? "extra-tools"
+      if build.with? "clang"
+        (buildpath/"tools/clang/tools/extra").install resource("extra-tools")
+      else
+        odie "clang's extra tools must be built with clang"
```

Comment:
You can leave it as-is. Sorry for creating confusion! We haven't agreed yet on whether `odie` or `raise` is preferable, as those cases are relatively rare. (We'll fix that ourselves later, if necessary.)